src/servers/CMakeLists.txt

```diff
@@ -164,6 +181,17 @@ add_executable(
   ${TRACING_OBJECTS}
 )
 set_property(TARGET main PROPERTY OUTPUT_NAME trtserver)
+
+if(${TRTIS_ENABLE_GPU})
+  target_include_directories(main PRIVATE ${CUDA_INCLUDE_DIRS})
+  target_link_libraries(
```

Comment:
Does main/trtserver now call some CUDA functions? Just passing around cudaIpcMemHandle_t doesn't require use to link in the cuda libraries.

Reply:
If I remove this I get the error: 'cudaIpcMemHandle_t' does not name a type during the build.

Reply:
During compile of a file? Or during link time?

Reply:
You're right. Removed.